hyperopt/pyll/base.py

```diff
@@ -319,7 +320,7 @@ def arg(self):
 
         fn = scope._impls[self.name]
         # XXX does not work for builtin functions
-        defaults = fn.__defaults__  # right-aligned default values for params
+        # defaults = fn.__defaults__  # right-aligned default values for params
```

Comment:
I commented it out since it's not used in this file after it's assigned. Maybe I should have been more careful at this point, but it's not clear to me where it's used. Will put it back in.